### PR TITLE
Adds new project

### PR DESCRIPTION
**Pull request by jnr0790**
updates new project for automation

### CODE REVIEW

File: .github/workflows/add-issue-to-cloud-project.yml [hunk 1]
```diff
@@ -15,5 +15,5 @@ jobs:
         uses: srggrs/assign-one-project-github-action@1.2.0
         if: github.event.action == 'opened'
         with:
-          project: "https://github.com/orgs/airbytehq/projects/3"
-          INITIAL_COLUMN_ID: "New"
+          project: "https://github.com/orgs/airbytehq/projects/23"
```
Comment (git-phu) [inline .github/workflows/add-issue-to-cloud-project.yml:18]: Is this the OC project?  Maybe we should update all the names in this workflow to say `Assign issue to OC Issues Project`

**Review by git-phu — COMMENTED** [review]
Awesome, thanks for updating!

Just had one comment

File: .github/workflows/add-issue-to-cloud-project.yml (hunk 1 above)
Comment (jnr0790) [reply .github/workflows/add-issue-to-cloud-project.yml:18]: Cool I'll do that.

Comment (jnr0790) [reply .github/workflows/add-issue-to-cloud-project.yml:18]: done.

**Review by git-phu — APPROVED** [review]

**Pull request merged**